rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -92,7 +92,7 @@ void cell_audio_config::reset(bool backend_changed)
 	audio_buffer_size = audio_buffer_length * audio_sample_size;
 
 	desired_buffer_duration = raw.desired_buffer_duration * 1000llu;
-	buffering_enabled = raw.buffering_enabled;
+	buffering_enabled = raw.buffering_enabled && backend->GetName() != "Null"; // Special case
```

Comment:
I'd prefer if the enabled state was set based on cfg options earlier

Reply:
Can you elaborate?

Reply:
in cellAudio.cpp line 536

Reply:
ah. no.
Actually just do this:
```suggestion
	buffering_enabled = raw.buffering_enabled && raw.renderer != audio_renderer::null;
```